### PR TITLE
[SPARK-11629] [ML] [PySpark] [Doc] Python example code for Multilayer Perceptron Classification

### PR DESCRIPTION
**Pull request by yanboliang**
Add Python example code for Multilayer Perceptron Classification, and make example code in user guide document testable. @mengxr

### CODE REVIEW

File: examples/src/main/java/org/apache/spark/examples/ml/JavaMultilayerPerceptronClassifierExample.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  public static void main(String[] args) {
+    SparkConf conf = new SparkConf().setAppName("JavaMultilayerPerceptronClassifierExample");
+    JavaSparkContext jsc = new JavaSparkContext(conf);
+    SQLContext jsql = new SQLContext(jsc);
+
+    // $example on$
+    // Load training data
+    String path = "data/mllib/sample_multiclass_classification_data.txt";
+    JavaRDD<LabeledPoint> data = MLUtils.loadLibSVMFile(jsc.sc(), path).toJavaRDD();
```
Comment (mengxr) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaMultilayerPerceptronClassifierExample.java]: Not part of this PR, but we should replace `MLUtils.loadLibSVMFile` by `sqlContext.read.format("libsvm").load(...)`. Could you submit another PR after this?

**Comment by mengxr** [comment]
LGTM. Merged into master and branch-1.6. Thanks!

File: examples/src/main/java/org/apache/spark/examples/ml/JavaMultilayerPerceptronClassifierExample.java (hunk 1 above)
Comment (yanboliang) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaMultilayerPerceptronClassifierExample.java]: Good point, I will send a PR.
